src/main/java/com/uid2/client/Envelope.java

```diff
@@ -0,0 +1,17 @@
+package com.uid2.client;
+
+
+public class Envelope {
```

Comment:
Should this go under `v2`/`v2api` namespace or otherwise specify what kind of envelope this is?

Reply:
calling this EnvelopeV2